app/components/scheduled_post_indicator/index.tsx

```diff
@@ -79,32 +79,28 @@ function ScheduledPostIndicator({
     );
 
     return (
-        <View
-            className='ScheduledPostIndicator'
-        >
```

Comment:
Not sure why there's a view with className.  But with newer version of expo-image, it's throwing a typescript error.